docs/user/icl/local-vc-authentication.inc

```diff
@@ -1,15 +1,63 @@
+|                      |                                                    | - potentially very insecure (when user has weak password)     |
+|                      |                                                    | - not available to all universities                           |
++----------------------+----------------------------------------------------+---------------------------------------------------------------+
+| User Token           | - easy to use, simple setup and revocation         | - dangerous if it gets leaked                                 |
+|                      | - useful for repository access via API             |                                                               |
+|                      | - very secure                                      |                                                               |
++----------------------+----------------------------------------------------+---------------------------------------------------------------+
+| Participation Token  | - easy to use, no setup needed                     | - only usable for one repository                              |
+|                      | - very limited access scope                        |                                                               |
+|                      | - very secure                                      |                                                               |
++----------------------+----------------------------------------------------+---------------------------------------------------------------+
+| SSH                  | - very secure                                      | - slower than HTTPS                                           |
+|                      | - industry standard for use with git               | - can be difficult to use                                     |
+|                      | - easy to use after correct set up                 | - can be overwhelming for new users                           |
++----------------------+----------------------------------------------------+---------------------------------------------------------------+
+
+The participation token is the preferred authentication method for students due to its simplicity, security, and lack of setup requirements.
+While SSH offers strong security, its complexity, setup challenges, and higher computational overhead make it less ideal.
+Password usage is discouraged for security reasons.
+Therefore, the order of authentication mechanisms should be ideally set up in the following order: token, SSH, and lastly password, with the option to disable password authentication entirely.
+
+
 Cloning a repository
 ^^^^^^^^^^^^^^^^^^^^
 
-You can use Sourcetree, git from the terminal, or any client you like to clone your Git repository.
+Using the chosen mechanism, you can use Sourcetree, git from the terminal, or any client you like to clone your Git repository.
```

Comment:
Can we get rid of SourceTree here and rather refer to Tower which is free for students? Please also add a link behind the name

Reply:
I added links to tower, and an overview on git clients